matchengine/engine.py

```diff
@@ -641,33 +811,36 @@ def _search_oncotree_diagnosis(onco_tree, c):
                 diagnoses = [diagnoses]
 
             for txt in diagnoses:
-                if txt.endswith("_LIQUID_") or txt.endswith("_SOLID_"):
+                if txt.endswith("_LIQUID_") or txt == 'All Liquid Tumors' or txt.endswith("_SOLID_") or txt == 'All Solid Tumors':
```

Comment:
Just a side not. We may have to come up with a list of tumor types that can be shared across institutions. `All Liquid Tumors` etc is  coming from OncoKB.